### ISSUE TITLE
BERT-like masked language pre-training?

### ISSUE THREAD

**Issue by JunhaoWang** [lucidrains/reformer-pytorch#62]
I'm pretty new to NLP, and trying to adapt reformer to do BERT-like masked language pretraining for long sequences. 
Is it as simple as setting causal=False in ReformerLM class ?

**Comment by lucidrains** [comment]
Yup! `causal = False` would allow all tokens to attend to each other
